test_stale_repositories.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/PuerkitoBio/goquery"
+	"golang.org/x/oauth2"
+)
+
+var reGithubRepo = regexp.MustCompile("https://github.com/[a-zA-Z0-9-.]+/[a-zA-Z0-9-.]+$")
```

Comment:
has a repository that can be outside the github, example:
- gitlab
- bitbucket
- ...

Reply:
hey @avelino. That is a good point. Should we restrict this to a set of possible sites (i.e. github, gitlab and bitbucket). I made use of the github api to check the commit age of a repository and there is an api. If we are handling all possible repository hosting sites how would we handle each specific one?